rbac/management/group/view.py

```diff
@@ -699,14 +712,14 @@ def principals(self, request: Request, uuid: Optional[UUID] = None):
                         service_accounts=service_accounts,
                         org_id=org_id,
                     )
-                new_principals = []
+                new_users = []
```

Comment:
Funny you mention that because I actually did that at first but I didn't want to be too invasive for long time devs on the project. But if you are okay with it, I think that would make the code more readable :D